src/app/clusters/ota-requestor/DefaultOTARequestor.cpp

```diff
@@ -522,22 +522,42 @@ void DefaultOTARequestor::TriggerImmediateQueryInternal()
     ConnectToProvider(kQueryImage);
 }
 
-OTARequestorInterface::OTATriggerResult DefaultOTARequestor::TriggerImmediateQuery()
```

Comment:
The previous enum seemed like it was semantic enough. Not sure why conversion to CHIP_ERROR

Reply:
Because most of the time we anyway want to convert it to `CHIP_ERROR` so it seems like just putting unnecessary burden on callers.